types/ranjs/index.d.ts

```diff
@@ -0,0 +1,569 @@
+// Type definitions for ranjs 1.22
+// Project: https://synesenom.github.io/ran/
```

Comment:
I think this library is exported as umd module under the name `ranjs`. If that's the case you need to add `export as namespace ranjs;`. This will make the global variable `ranjs` accessible.

Reply:
Thanks! I have added the changes.